lib/src/components/NxTreeView/NxTreeView.scss

```
@@ -18,16 +18,19 @@ $twisty-proportion-difference: 2px;
 */
 .nx-tree-view {
   box-sizing: border-box;
-  margin: var(--nx-spacing-4x) 0;
+  // left gets 7px which keeps the caret from overhanging the container
+  margin: var(--nx-spacing-4x) calc(0px - var(--nx-spacing-2x)) var(--nx-spacing-4x) calc(1px - var(--nx-spacing-2x));
```

I think this fully reflects what's going on here:
```suggestion
  margin: var(--nx-spacing-4x) calc(0px - var(--nx-spacing-2x)) var(--nx-spacing-4x) calc(#{$twisty-proportion-difference / 2} - var(--nx-spacing-2x));
```

---

> I think this fully reflects what's going on here:

Checking again, this isn't right.  If you look at the other usages of `$twisty-proportion-difference`, you'll see that it's already intended to be the value on one side of the twisty, not the sum of both.  So we shouldn't be dividing by 2 here.  

At the same time, it makes me wonder why using that variable here is necessary at all – the twisty already gets its own extra padding using this variable when it is expanded, which should take care of this situation.  I'm thinking maybe the variable needs to be 3, not 2.  Or maybe 2.5.  The true difference between the height at width of the `path` is 5.53px.

---

> > I think this fully reflects what's going on here:
> 
> Checking again, this isn't right. If you look at the other usages of `$twisty-proportion-difference`, you'll see that it's already intended to be the value on one side of the twisty, not the sum of both. So we shouldn't be dividing by 2 here.
> 
> At the same time, it makes me wonder why using that variable here is necessary at all – the twisty already gets its own extra padding using this variable when it is expanded, which should take care of this situation. I'm thinking maybe the variable needs to be 3, not 2. Or maybe 2.5. The true difference between the height and width of the `path` is 5.53px.

So the upshot is that we'll leave this as-is?

---

> So the upshot is that we'll leave this as-is?

No, the upshot is that we shouldn't have had to do anything special about the twisty on this line as the styles it already had were supposed to be handling the problem we're having here.  They aren't, because the value of that variable is wrong.

---

> > So the upshot is that we'll leave this as-is?
> 
> No, the upshot is that we shouldn't have had to do anything special about the twisty on this line as the styles it already had were supposed to be handling the problem we're having here. They aren't, because the value of that variable is wrong.

Based on the `5.53px` value you mentioned above I set the variable to `2.75px`. Zoomed in very close on my monitor it looks perfect. https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/3e3535b08a1d9720a57ee7078f83f0a14de37d4d